Project.toml

```diff
@@ -2,7 +2,7 @@ name = "CodecZstd"
 uuid = "6b39b394-51ab-5f42-8807-6242bab2b4c2"
 license = "MIT"
 authors = ["Kenta Sato <bicycle1885@gmail.com>"]
-version = "0.6.0"
+version = "0.6.1"
```

Comment:
Why do you change the version? Is this required?

Reply:
Ah, I just realized that 0.6.0 hasn't been tagged yet. I was working under the assumption that 0.6.0 had already been tagged, in which case this would allow the next version to be 0.6.1. I can drop this commit.

Reply:
I'm waiting for https://github.com/JuliaRegistries/General/pull/2440 to be merged ;) Anyway, thank you for dropping it because I want to separate merging pull requests and releasing a new version.

Reply:
It's merged now 😄

Reply:
what a coincidence :+1: